Add Usage with PHPStan to README (#15)
GiacoCorsiglia/wordpress-stubs: README.md

If your editor has trouble parsing all of WordPress, you may find the stubs useful for enabling code completion and related features.  For example, [here](https://github.com/bmewburn/vscode-intelephense/issues/113) are instructions for usage with VSCode's [Intelephense](https://marketplace.visualstudio.com/items?itemName=bmewburn.vscode-intelephense-client) extension.  (In my experience, however, Intelephense parses WordPress core just fine.)

## Usage with PHPStan

```bash
composer require --dev szepeviktor/phpstan-wordpress
```

> This package depends on PHPStan and WordPress Stubs.

Reviewer: @szepeviktor 

 What do you think about adjusting so it's easier for interested users to navigate to these packages?

```suggestion

The package [`szepeviktor/phpstan-wordpress`](https://github.com/szepeviktor/phpstan-wordpress) depends on [`phpstan/phpstan`](http://github.com/phpstan/phpstan) and `giacocorsiglia/wordpress-stubs`.
```

Author: Yes!
